M2/Macaulay2/c/error.c

```diff
@@ -117,11 +117,11 @@ void fail(char *filename, int lineno) {
 void downpos(node n){
      struct POS *p = pos(n);
      if (p != NULL && p->filename != NULL) {
-	  fprintf(stderr,errfmt,p->filename,p->lineno,p->column+1,"");
+	  fprintf(stderr,errfmt,p->filename,p->lineno,p->column,"");
```

Comment:
What does this do again?

Reply:
you mean the +1? it was part of the numerous hacks to circumvent the inaccuracy of code location. no longer needed.